components/locid/src/extensions/unicode/attributes.rs

```diff
@@ -91,7 +92,7 @@ impl std::fmt::Display for Attributes {
             if initial {
                 initial = false;
             } else {
-                f.write_str("-")?;
+                f.write_char('-')?;
```

Comment:
Just curious, why is this different than write_str for "-" case?

Reply:
It's just an inch cheaper - https://github.com/rust-lang/rust/pull/24661